internal/executor/kubectl/executor.go

```diff
@@ -3,12 +3,20 @@ package kubectl
 import (
```

Comment:
1. It's a different thing. The plugin is a binary with stdout and stderr, we log the stdout only if plugin debug is enabled. However the plugin itself can log different things to stdout and this can be just a business logic of a given plugin.
2. The `helm` plugin already supports the default namespace. I don't have a strong opinion about having the `defaultNamespace` under `context` or `config`. For now, we don't have RBAC yet, so maybe let's wait until we will have it and the we can replace it globally 👍